### ISSUE TITLE
NSFPlug-generated playlists not formatted correctly

### ISSUE THREAD

**Issue by KyleJCrb** [bbbradsmith/nsfplay#56]
When using NSFplug's playlist generator (under File/Winamp Playlist,) the resulting m3u file does not format itself correctly. It seems to pull the title formatting from the main settings rather than using the NSFe information present to generate the playlist. I've attached an example of how it looks below. This playlist does play fine within Winamp/WACUP, but of course all of the song information is incorrect.

```
[playlist]
File1=F:\Music\Game\Original Soundtracks\Proprietary Format\NSFe\NSFe_Collection_Part1_NTSC\Danny Sullivan's Indy Heat.nsfe::NSF,$01,Title Screen (001) - Danny Sullivan's Indy Heat,,
File2=F:\Music\Game\Original Soundtracks\Proprietary Format\NSFe\NSFe_Collection_Part1_NTSC\Danny Sullivan's Indy Heat.nsfe::NSF,$02,Title Screen (001) - Danny Sullivan's Indy Heat,,
File3=F:\Music\Game\Original Soundtracks\Proprietary Format\NSFe\NSFe_Collection_Part1_NTSC\Danny Sullivan's Indy Heat.nsfe::NSF,$03,Title Screen (001) - Danny Sullivan's Indy Heat,,
File4=F:\Music\Game\Original Soundtracks\Proprietary Format\NSFe\NSFe_Collection_Part1_NTSC\Danny Sullivan's Indy Heat.nsfe::NSF,$04,Title Screen (001) - Danny Sullivan's Indy Heat,,
File5=F:\Music\Game\Original Soundtracks\Proprietary Format\NSFe\NSFe_Collection_Part1_NTSC\Danny Sullivan's Indy Heat.nsfe::NSF,$05,Title Screen (001) - Danny Sullivan's Indy Heat,,
File6=F:\Music\Game\Original Soundtracks\Proprietary Format\NSFe\NSFe_Collection_Part1_NTSC\Danny Sullivan's Indy Heat.nsfe::NSF,$06,Title Screen (001) - Danny Sullivan's Indy Heat,,
File7=F:\Music\Game\Original Soundtracks\Proprietary Format\NSFe\NSFe_Collection_Part1_NTSC\Danny Sullivan's Indy Heat.nsfe::NSF,$07,Title Screen (001) - Danny Sullivan's Indy Heat,,
File8=F:\Music\Game\Original Soundtracks\Proprietary Format\NSFe\NSFe_Collection_Part1_NTSC\Danny Sullivan's Indy Heat.nsfe::NSF,$08,Title Screen (001) - Danny Sullivan's Indy Heat,,
File9=F:\Music\Game\Original Soundtracks\Proprietary Format\NSFe\NSFe_Collection_Part1_NTSC\Danny Sullivan's Indy Heat.nsfe::NSF,$09,Title Screen (001) - Danny Sullivan's Indy Heat,,
File10=F:\Music\Game\Original Soundtracks\Proprietary Format\NSFe\NSFe_Collection_Part1_NTSC\Danny Sullivan's Indy Heat.nsfe::NSF,$0a,Title Screen (001) - Danny Sullivan's Indy Heat,,
File11=F:\Music\Game\Original Soundtracks\Proprietary Format\NSFe\NSFe_Collection_Part1_NTSC\Danny Sullivan's Indy Heat.nsfe::NSF,$0b,Title Screen (001) - Danny Sullivan's Indy Heat,,
File12=F:\Music\Game\Original Soundtracks\Proprietary Format\NSFe\NSFe_Collection_Part1_NTSC\Danny Sullivan's Indy Heat.nsfe::NSF,$0c,Title Screen (001) - Danny Sullivan's Indy Heat,,
File13=F:\Music\Game\Original Soundtracks\Proprietary Format\NSFe\NSFe_Collection_Part1_NTSC\Danny Sullivan's Indy Heat.nsfe::NSF,$0d,Title Screen (001) - Danny Sullivan's Indy Heat,,
File14=F:\Music\Game\Original Soundtracks\Proprietary Format\NSFe\NSFe_Collection_Part1_NTSC\Danny Sullivan's Indy Heat.nsfe::NSF,$0e,Title Screen (001) - Danny Sullivan's Indy Heat,,
NumberOfEntries=14
Version=2

```

**Comment by bbbradsmith** [comment]
I've never understood how to use this feature. I did not touch the relevant code when I added NSFe support, so I guess it's still doing whatever it always did.

So, what you're expecting this to do is provide a way to convert NSFe metadata into a .pls file?

**Comment by bbbradsmith** [comment]
It looks like the playlist generator had a bug where switching songs didn't update the title. This should work correctly now, and output the title string for each track. If you'd like the title string to use the NSFe's track name, you can edit its formatting to do so in the main options panel.

Should be fixed by: 565d19c67dc464b1c80475479bb6a10ed388871b
